01 - Community/Obsidian Roundup/2022-01-22 API improvements for devs & Reference vs. Reflection usage.md

```diff
@@ -151,4 +151,10 @@ With your help, I can.
 
 PS: Subscription prices gets locked at the moment you sign up. It's possible that I'll increase the price of new subscriptions if I start doing things like office hours, or weekly essays instead of monthly.
 
-& don't worry — even if I do have to go back into the school building, this newsletter will continue to exist. The Roundup itself will always be free.
\ No newline at end of file
+& don't worry — even if I do have to go back into the school building, this newsletter will continue to exist. The Roundup itself will always be free.
```

Comment:
These changes are unrelated to the file the "Issues requesting help" file that this PR is about.

Reply:
> I'll close this PR as it doesn't change the file that the script is maintaining - but it does change a bunch of unrelated files.
> 
> I'll have a look at the script and see if it can be changed to only create a PR if there are any changes to the file containing issues requesting help.

Done in https://github.com/obsidian-community/obsidian-hub/pull/373